### DISCUSSION TITLE
怀疑PID大作业平台答案有问题

### DISCUSSION THREAD

**Discussion by AuroraRyan** [physics-data/faq#173] [Q&A] [answered]
怀疑答案有些问题，对测试集用LSTM拟合有较好的学习曲线
![微信图片_20210921003344](https://user-images.githubusercontent.com/88898895/134040107-12fc6fa7-e126-4df3-a393-e18bf520853d.png)
将测试集答案随机排列打散后，无学习效果
![微信图片_20210921003744](https://user-images.githubusercontent.com/88898895/134040281-c6681b5e-45b6-45dc-b768-29b7ccb684bc.png)
直接用问题集（problem.h5）和问题集答案（submit.h5）进行拟合，无学习效果
![微信图片_20210921003754](https://user-images.githubusercontent.com/88898895/134040432-3593d429-2ed0-4614-9e73-be2fc7e15a39.png)
据此怀疑答案（submit.h5）与问题集（problem.h5）不匹配
以上图片纵轴标签有些错误，均为准确率，而不是错误率，但并不影响学习效果判断

**Comment by heroxbd** [comment, marked as answer]
我在 leaderboard 上看到了同样的问题，正在调查。

更新：非常抱歉，平台上的答案有问题，在 2021年9月21日22:45 完成了更新，请重试。

@wengjun232
